### ISSUE TITLE
How do you correctly use the CSS?

### ISSUE THREAD

**Issue by karlmacklin** [react-toolbox/react-toolbox#231]
My question might stem from too much of a newbie perspective, please ignore in that case.
Anyways - I find it very hard to get started using these components.

I follow the instructions and start out with the react-hot-boilerplate, and then add the needed loaders for webpack etc.

I have installed the react-toolbox via npm. Add Roboto font and material ui icons.
Trying out a few examples, I can get components to show up on screen but they are not styled correctly. Everything is much too big, and wrong font etc.
From some deduction I realize that the examples online are all shown with styles from the common.scss file.

What is the correct way to import these styles, with or without your own theme? Copy them all form the node_modules folder and put in your own project? Or trigger them some other way? There's a toolbox-loader that specifies that webpack config option 'toolbox' can be used to define your theme.scss filename, but it seems that whatever you name it there it always looks for exactly theme.scss in your project root folder.

So far my solution is to copy the first style-chunk on the generated html on the live examples (the one that includes normalize etc), which is not a pretty solution.

Any hints?

I realize the sizing problem I have is related to the 62.5% font-size mentioned in this issue as well: https://github.com/react-toolbox/react-toolbox/issues/172 - but I'm not sure how far this bug affects my starting problems.

**Comment by Bartekus** [comment]
If you fork the whole project and install it as per instruction you'll end up with a react-toolbox folder and in it you'll find /docs folder which is an excellent start point of learning, it also is based on https://github.com/gaearon/react-transform-boilerplate which is much better than react-hot-boilerplate.

**Comment by javivelasco** [comment]
Hi @karlmacklin !!

To use react toolbox you'd need to set up a project with webpack, sass loader with css modules and toolbox loader if you want to customize colors and sizes. Also you'd need to add in your index Roboto fond and Material icons with a `link` tag since it's not a good practice to provide them in the package css. 

As @Bartekus says (thanks for your answer!), the `docs` folder, which is building the documentation, uses react toolbox and it's a good starting point.  Also, a very important point that probably should be better documented is the `commons`. We are adding in that file some resets for the css and setting up a root font size of `10px`. If you don't include commons by importing `import 'react-toolbox/lib/commons'`, you'd need to set up the resets manually and also set a `$unit` base size adjusting your needs. For example, changing `$unit` in your sass configuration file with `10px` unit would be enough.

Hope you get to configure it. Please, let us know to close the issue as soon as you get it. Let's keep it open for a few days in case somebody wants to point anything else.

Thanks! :)

**Comment by karlmacklin** [comment]
Looking at the docs example - do I understand it correctly that when not using a custom theme it's the _globals.scss that represents the main styles (primary/secondary color etc) that would be suitable to overwritein your own theme.scss?

I got thrown off by the small issue of not being able to specify a different theme.scss in the toolbox-loader, I think that might be a bug. Cannot change.

Also didn't see the import 'react-toolbox/commons'; in app/index.jsx so couldn't (until now) figure out where commons was loaded from.

Feel free to close this whenever, I've sorted it out! :)
